Frameworks/UIKit/UIViewController.mm

```diff
@@ -2148,6 +2103,17 @@ - (UIModalPresentationStyle)modalPresentationStyle {
 }
 
 /**
+ @Status Caveat
+ @Notes  WinObjC-only extension providing access
+         to UIModalPresentationFormSheet overlay.
+         Not exposed in header (clients must access
+         via performSelector).
+*/
+- (UIView*)_modalOverlayView {
+    return priv->_modalOverlayView;
+}
```

Comment:
sounds good - i'm about to pull your changes and try them out locally

---
In reply to: [109035577](https://github.com/Microsoft/WinObjC/pull/2195#discussion_r109035577) [](ancestors = 109035577)